src/Extensions/Compatibility/TimedScopes/TimedScope.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System;
+using System.Diagnostics;
+using Microsoft.Omex.Extensions.Abstractions.Activities;
+
+namespace Microsoft.Omex.Extensions.Compatibility
+{
+	/// <summary>
+	/// Extension for string to start Activity
+	/// </summary>
+	public static class TimedScope
```

Comment:
>TimedScope [](start = 21, length = 10)

What is the reason for keeping the name?

Reply:
Its compatibility package, which is intended to be used as an intermediate step for switching,
So I thought that keeping the have would help people understand what should be used as timescopes replacement.
I'm also considering just deleting these packages since it's not proving very useful.
I'm open to suggestions about naming

Reply:
I am more on the side of not having a compatibility package not to increase the complexity, but it will be interesting to see other opinions as well.

---
In reply to: [561022206](https://github.com/microsoft/Omex/pull/285#discussion_r561022206) [](ancestors = 561022206)

Reply:
Kind of agree with Daniel here on dropping TimedScopes.
If the shim-layer enabled existing dependent applications running in production to soft-migrate without breaking any metrics, dashboards or alerting I'd think it was a good idea but that is not the case here.

Having a shim-layer that doesn't fix the issues caused by this change may cause developers to assume the change does not have the drastic cutover effects it does.

Reply:
Less code - less problems, deleted Compatability packages completely :)